package.json

```diff
@@ -50,6 +50,7 @@
   },
   "dependencies": {
     "@testing-library/dom": "^6.1.0",
+    "@testing-library/jest-dom": "^4.2.4",
```

Comment:
with package.json changes we need to follow with updating yarn.lock file. that is the reason the PR's build broke.